pkg/cluster/manager/display.go

```diff
@@ -844,9 +844,5 @@ func (m *Manager) displayDashboards(ctx context.Context, t *spec.Specification,
 		}
 	}
 
-	if m.logger.GetDisplayMode() != logprinter.DisplayModeJSON || j == nil {
-		fmt.Printf("Dashboard URLs:     %s\n", strings.Join(dashboardAddrs, ","))
```

Comment:
It is a new function that can display multiple dashboard address(dashboard_servers). So it is in valid to remove it.

Reply:
Maybe there should be an `if len(dashboardAddrs) > 1` around it instead? And then also a `if len(dashboardAddrs) == 1`  before line 836 ?

Also, does it use the right `scheme` if there are multiple?

Reply:
PTAL @xhebox

Reply:
Hmm, LGTM. But I don't know if it is wanted or not.

It is a new feature but `Dashboard URL: xxx` is an old feature. There are compatibility issues.

This feature is asked by the team of @sydpz, PTAL.

Reply:
How about displaying multiple dashboard URLs in `Dashboard URL`?

Reply:
> How about displaying multiple dashboard URLs in `Dashboard URL`?

I've considered, and I think it will break existed applications. For example, the team of TEM does rely on that.

Reply:
In the output or in the json structure?

Reply:
Output, of course. I did not modify json structure, iirc.

Reply:
Shouldn't humans rely on the human readable output and programs on the machine readable json? Maybe even over a REST interface instead of by calling a command.

Shouldn't TEM be modified to follow the output of tiup?

Another option would be to do something like git with Porcelain and Plumbing. Or something like `--output v1`...